src/libcommon/utility/types.h

```diff
@@ -26,6 +26,7 @@
 #include <sstream>
 #include <unordered_set>
 #include <variant>
+#include <qdebug.h>
```

Comment:
Usefull?

Reply:
It is needed to get the same beahviour with side logs, ie:
`qCDebug(lcCommClient()) << "Snd rqst" << id << num;`

line 497:
```
inline QDebug &operator<<(QDebug &os, C e) {
    return os << toStringWithCode(e).c_str();
}
```